Essentials/Commands/CleanupModule.cs

```diff
@@ -59,6 +59,30 @@ public void Delete()
             Log.Info($"Cleanup deleted {count} grids matching conditions {string.Join(", ", Context.Args)}");
         }
 
+        [Command("delete offtype", "Deletes grids with specified blocks toggled off.")]
```

Comment:
Why is this a separate command tho?

I personally prefer it to keep it as !cleanup list or !cleanup delete

Instead you could have added a new condition you can see how for example HasType is implemented in line 385

As it is implemented now I have not the ability to just "list" which grids would be affected. And since I cannot see what would be affected I cannot check if my filter is correct or I for example need to exclude something with other conditions.